src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Core/Packaging/PluginPackage.cs

```diff
@@ -45,69 +51,19 @@ public abstract class PluginPackage
         public abstract IReadOnlyCollection<PluginPackageEntry> Entries { get; }
 
         /// <summary>
-        ///     Gets the plugin metadata.
-        /// </summary>
-        public PluginMetadata PluginMetadata { get; }
-
-        /// <summary>
-        ///     Gets the plugin identity.
-        /// </summary>
-        public PluginIdentity PluginIdentity
-        {
-            get
-            {
-                return this.PluginMetadata.Identity;
-            }
-        }
-
-        /// <summary>
-        ///     Gets the plugin ID.
+        ///     Gets the plugin information.
         /// </summary>
-        public string Id
-        {
-            get
-            {
-                return this.PluginIdentity.Id;
-            }
-        }
+        public PluginInfo PluginInfo { get; }
 
         /// <summary>
-        ///     Gets the plugin version.
+        ///     Gets the plugin contents.
         /// </summary>
-        public Version Version
-        {
-            get
-            {
-                return this.PluginIdentity.Version;
-            }
-        }
-
-        /// <summary>
-        ///     Gets the plugin display name.
-        /// </summary>
-        public string DisplayName
-        {
-            get
-            {
-                return this.PluginMetadata.DisplayName;
-            }
-        }
-
-        /// <summary>
-        ///     Gets the plugin description.
-        /// </summary>
-        public string Description
-        {
-            get
-            {
-                return this.PluginMetadata.Description;
-            }
-        }
+        public PluginContents PluginContents { get; }
```

Comment:
Would it be better to call it `PluginContentsInfo` or something? I feel like the `contents` of a package sounds more like all the actual files within the package. And in `AvailablePlugin.cs` we have this `GetPluginContents` which sounds to me like we are getting the binaries of the plugin.

Reply:
Sure